### DISCUSSION TITLE
About Icon of NeoForge

### DISCUSSION THREAD

**Discussion by MoYuan-CN** [neoforged/NeoForge#1155] [Q&A]
Is there an Icon for NeoForge has been painted? I want to use a icon in my launcher like this:
![image](https://github.com/neoforged/NeoForge/assets/109946426/62b63a06-e4cf-4178-9368-41117337fa3a)

**Comment by sciwhiz12** [comment]
Our official branding and art, including the NeoForge fox icon, is present in [our `.github` repository](https://github.com/neoforged/.github/tree/main/art). 

Unforunately, we neither have a PNG of the icon itself (only an ICO) or a black-and-white version suitable for the interface you screenshotted.

**Reply by Gaming32** [reply]
There is an unofficial icon, used by Modrinth, in this style, however.

**Reply by sciwhiz12** [reply]
Ah yes, I believe I saw it mentioned/shown briefly in the NeoForge Discord when it was made, though I don't know where it is currently stored.
